PayGreen/paygreen-ui
# PGX-4453 - Nouveau composant Overlay

## How to test ?

- Launch the storybook
- Go to section 'Overlay'
- Check the gradient color and direction, and its opacity
